.github/workflows/build.yml

```
@@ -0,0 +1,88 @@
+name: Build app package
+
+on: [ push, pull_request, workflow_dispatch ]
+
+jobs:
+  build_pkg:
+    runs-on: ubuntu-latest
```

You should use a hardcoded version to avoid unexpected breakage.
```suggestion
    runs-on: ubuntu-20.04
```